legacy/messaging/src/main/java/org/jboss/as/messaging/MessagingTransformers.java

```diff
@@ -116,10 +87,21 @@ static void registerTransformers(final SubsystemRegistration subsystem) {
      */
     private static void buildTransformers2_1_0(ResourceTransformationDescriptionBuilder builder) {
         ResourceTransformationDescriptionBuilder hornetqServer = builder.addChildResource(pathElement(HORNETQ_SERVER));
-        rejectDefinedAttributeWithDefaultValue(hornetqServer, OVERRIDE_IN_VM_SECURITY);
-
         ResourceTransformationDescriptionBuilder addressSetting = hornetqServer.addChildResource(AddressSettingDefinition.PATH);
-        rejectDefinedAttributeWithDefaultValue(addressSetting, MAX_REDELIVERY_DELAY, REDELIVERY_MULTIPLIER, SLOW_CONSUMER_CHECK_PERIOD, SLOW_CONSUMER_POLICY, SLOW_CONSUMER_THRESHOLD);
+        rejectDefinedAttributeWithDefaultValue(addressSetting, MAX_REDELIVERY_DELAY, REDELIVERY_MULTIPLIER);
+
+        ResourceTransformationDescriptionBuilder bridge = hornetqServer.addChildResource(BridgeDefinition.PATH);
+        bridge.getAttributeBuilder().setValueConverter(new DoubleToBigDecimalConverter(), RETRY_INTERVAL_MULTIPLIER);
```

Comment:
As one of last commits in messaging subsystem changed the type of attribute from big decimal to double.
see https://github.com/wildfly/wildfly/commit/825187ca806bc34fa4bbbd184cd098651b3c8a19#diff-8ba799e6861eb2ccc8ce2ef54a6de31eL167